File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        
        // Used HashSet for unique values
        HashSet<Integer> set = new HashSet<Integer>();

        for(int i = 0 ; i < nums.length ; i++) {

            // If HashSet contains means value exist and it contain duplicate value
            if(set.contains(nums[i])) {
               return true; 
            }
            set.add(nums[i]);
        }
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.kt
class Solution {
    fun containsDuplicate(nums: IntArray): Boolean {
        val hashSet = mutableSetOf<Int>()

        for(i in nums.indices) {
            if(hashSet.contains(nums[i])) {
                return true
            }
            hashSet.add(nums[i])
        }
    
        return false
    }

}